manageiq-operator/pkg/controller/manageiq/manageiq_controller.go

```diff
@@ -299,9 +324,9 @@ func (r *ReconcileManageiq) generateRbacResources(cr *miqv1alpha1.Manageiq) erro
 	return nil
 }
 
-func (r *ReconcileManageiq) createk8sResIfNotExist(cr *miqv1alpha1.Manageiq, res, restype metav1.Object) error {
+func (r *ReconcileManageiq) createk8sResIfNotExist(owner, res, restype metav1.Object) error {
```

Comment:
Why change this? I still only see "Manageiq" structs being passed as this parameter?